lib/core.js

```diff
@@ -194,6 +195,19 @@ module.exports = class Core {
     return new this(header, compat, crypto, oplog, bigHeader, tree, blocks, bitfield, verifier, legacy, opts.onupdate || noop, opts.onconflict || noop)
   }
 
+  async audit () {
+    await this._mutex.lock()
+
+    try {
+      await this._flushOplog()
```

Comment:
Not a comment, but a question: in which case does the oplog need flushing here, or is it more of a just-in-case sanity check?

Reply:
cause we load it all from the files in audit